models/profileModel.js

```diff
@@ -0,0 +1,27 @@
+const mongoose = require('mongoose');
+
+//Structure in the collection
+const Schema = mongoose.Schema;
+
+//https://mongoosejs.com/docs/guide.html
+//Schema structure how to save in the collection
+//Objects with a string and all are required
+const profileSchema = new Schema({
+  name: {
+    type: String
+  },
+  photo: {
+    type: String
+  },
+  age: {
+    type: Number
```

Comment:
```suggestion
    type: Number
    min: 18
```

Nice to put a min age of 18 on our matching app